README.md

```diff
@@ -7,22 +7,24 @@
 ClusterFuzz is a scalable [fuzzing](https://en.wikipedia.org/wiki/Fuzzing)
 infrastructure that finds security and stability issues in software.
 
-Google uses ClusterFuzz to fuzz the Chrome Browser and as the fuzzing
+Google uses ClusterFuzz to fuzz all Google products and as the fuzzing
 backend for [OSS-Fuzz].
 
 ClusterFuzz provides many features which help seamlessly integrate fuzzing into
 a software project's development process:
-- Highly scalable. Google's internal instance runs on over 25,000 machines.
+- Highly scalable. Can run on any size cluster (e.g. Google's instance runs on
+  30,000 VMs).
 - Accurate deduplication of crashes.
-- Fully automatic bug filing and closing for issue trackers
-  ([Monorail] only for now).
+- Fully automatic bug filing, triage and closing for various issue trackers
+  (e.g. [Monorail], [Jira]).
+- Support for coverage guided fuzzing (e.g. [libFuzzer], [AFL] and [Honggfuzz]).
+  Ensembles fuzzers for optimal fuzzing efficiency.
```

Comment:
I would say: "Supports multiple engines (e.g. Honggfuzz, AFL, LibFuzzer) for optimal results (ensemble fuzzing)".
The term "ensemble fuzing" isn't understandable to people who haven't read the paper.

Reply:
Done!